test/deal-investor/get-deal-investors-by-deal-identifier.api-test.ts

```diff
@@ -0,0 +1,146 @@
+import { UkefId } from '@ukef/helpers';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+import { DealInvestorGenerator } from '../support/generator/deal-investor-generator';
+
+describe('GET /deals/{dealIdentifier}/investors', () => {
+  const valueGenerator = new RandomValueGenerator();
+  // TODO: Get portfolio default value from default constants;
+  const dealIdentifier: UkefId = valueGenerator.ukefId();
+  const portfolioIdentifier = 'E1';
```

Comment:
Constant please.

Reply:
Changed to Constant because it was introduced by other PR. Removed TODO.